### PR TITLE
#76: Assertions wrapped in a count(*) query instead of selecting actual rows

### PR DESCRIPTION
**Pull request by ShevtsovY**

### CODE REVIEW

File: api/commands/run.ts [hunk 1]
```diff
@@ -144,8 +144,9 @@ export class Runner {
             const rows = await this.adapter.execute(task.statement, handleCancel =>
               this.eEmitter.on(CANCEL_EVENT, handleCancel)
             );
+            const rowsNumber = !!rows.length ? rows[0].rowsNumber : 0;
```
Comment (BenBirt) [inline api/commands/run.ts]: rowCount?

File: core/adapters/bigquery.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class BigQueryAdapter extends Adapter implements IAdapter {
       name: a.name
     });
     tasks.add(Task.statement(this.createOrReplaceView(assertionTarget, a.query)));
-    tasks.add(Task.assertion(`select 1 as detect from ${this.resolveTarget(assertionTarget)}`));
+    tasks.add(Task.assertion(`select count(*) as rows_number from ${this.resolveTarget(assertionTarget)}`));
```
Comment (BenBirt) [inline core/adapters/bigquery.ts]: rowCount (in the other queries too please)

File: api/commands/run.ts (hunk 1 above)
Comment (ShevtsovY) [reply api/commands/run.ts]: ok

File: core/adapters/bigquery.ts (hunk 2 above)
Comment (ShevtsovY) [reply core/adapters/bigquery.ts]: ok

**Pull request merged**